en/includes/guides/organization-management/cleanup-organization-resources.md

```diff
@@ -0,0 +1,10 @@
+# Resources Cleanup for Deleted Organizations
+
+When you delete an organization in WSO2 Identity Server, its associated data remains in the database. To permanently remove this data, use the cleanup scripts provided by WSO2.
+
+## Run the cleanup
+
```

Comment:
Add a recommendation stating that this should be executed in off peak hours in production systems